workflows/metatdenovo.nf

```diff
@@ -73,6 +79,10 @@ include { DIGINORM } from '../subworkflows/local/diginorm' addParams(
 def multiqc_options   = modules['multiqc']
 multiqc_options.args += params.multiqc_title ? Utils.joinModuleArgs(["--title \"$params.multiqc_title\""]) : ''
 
+params.prodigal_trainingfile = file("../test_prodigal/training_file.trn")
```

Comment:
This doesn't belong here, should be set in the relevant config file, e.g. `conf/test.config`.